include/zephyr/devicetree/clocks.h

```diff
@@ -48,7 +48,8 @@ extern "C" {
  *
  * @param node_id node identifier; may or may not have any clocks property
  * @param idx index of a clocks property phandle-array whose existence to check
- * @return 1 if the index exists, 0 otherwise
+ * @retval 1 if the index exists
+ * @retval 0 otherwise
```

Comment:
one would typically not include the leading "If..." (I know it's poorly enforce in-tree _currently_), also, since this is effectively boolean maybe:



```suggestion
 * @retval 1 a clock is present at index @p idx
 * @retval 0 no clock is present at index @p idx
```